### PR TITLE
Add the necessary signal connect and disconnect to fix issues while saving the fixtures

### PR DESCRIPTION
**Pull request by schenedx**
This is a PR to fix a bug, where the program fixture loading management command was broken when new signals were introduced.
See the proof of fix at https://tools-edx-jenkins.edx.org/job/Sandboxes/job/UpdateMastersSandbox/

### CODE REVIEW

File: course_discovery/apps/course_metadata/management/commands/load_program_fixture.py [hunk 1]
```diff
@@ -65,6 +75,12 @@ def disconnect_program_signals():
             publish_masters_track, sender=Seat,
         )
 
+        pre_save.connect(
+            check_curriculum_for_cycles, sender=Curriculum,
+        )
+        pre_save.connect(
+            check_curriculum_program_membership_for_cycles, sender=CurriculumProgramMembership,
+        )
         pre_save.connect(
```
Comment (kdmccormick) [inline course_discovery/apps/course_metadata/management/commands/load_program_fixture.py]: Why not have a list of tuples in the form `(signal_name, sender_model_class)` and then iterate through that to disconnect and re-connect? That way you know you that you re-connect everything you disconnected.

**Review by kdmccormick — COMMENTED** [review]
Were you able to get that test working to ensure that new signals cause a test failure?

File: course_discovery/apps/course_metadata/management/commands/load_program_fixture.py (hunk 1 above)
Comment (schenedx) [reply course_discovery/apps/course_metadata/management/commands/load_program_fixture.py]: Good suggestion for optimization. I will try that.

File: course_discovery/apps/course_metadata/management/commands/load_program_fixture.py [hunk 2]
```diff
@@ -32,48 +34,61 @@ def disconnect_program_signals():
...
-    pre_save.disconnect(
-        ensure_external_key_uniquness__course_run, sender=CourseRun,
-    )
-    pre_save.disconnect(
-        ensure_external_key_uniquness__curriculum, sender=Curriculum,
-    )
-    pre_save.disconnect(
-        ensure_external_key_uniquness__curriculum_course_membership, sender=CurriculumCourseMembership,
-    )
+    signals_list = [
```
Comment (kdmccormick) [inline course_discovery/apps/course_metadata/management/commands/load_program_fixture.py:46]: Nice!

**Review by zacharis278 — APPROVED** [review]
this lgtm

**Comment by schenedx** [comment]
@kdmccormick I got the test to fail before the change. The key is to have fixture file include `CurriculumProgramMembership` before `Curriculum`. This way, the signal would cause the failure during fixture file save. Once this build is green, I will merge.

**Pull request merged**